### DISCUSSION TITLE
Documentation lack some critical information for beginners, isn't?

### DISCUSSION THREAD

**Discussion by ArKam** [ory/kratos#4060] [Q&A]
Hi everyone,

I'm wondering if I didn't find it or if some documentation wasn't missing because when trying to run services such as kratos for instance, there is no information about how to configure it properly.

I'm using ory/kratos helm chart to deploy it on a kubernetes cluster, however, it doesn't really work out of the box as the default kratos configuration lack some sane defaults.

Indeed, when launching the helm chart with default installation process, the instance isn't able to find out few required informations such as a data source name for the database (Should probably default to a embedded SQLi instance by default) or yet schemas for the identity endpoints.

Now, I found out few fragments of information [here](https://github.com/ory/kratos/blob/master/quickstart-postgres.yml) and [there](https://www.youtube.com/watch?v=F6ZKrxf8LuQ)

But except if I missed it and in that case I do sincerely apology, the OSS documentation is lacking a bit of default/beginners settings informations.

Thanks everyone!

**Comment by Edregol** [comment]
Have you tried the documentation on their website? https://www.ory.sh/docs/welcome

**Comment by vinckr** [comment]
Hey @ArKam 

thanks for the feedback. I would recommend to try out the quickstarts before the helm charts: https://www.ory.sh/docs/kratos/quickstart
There are also more community examples here: https://github.com/ory/examples/
